docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -6,8 +6,8 @@
 # Combobox
```

Comment:
while I don't disagree. I often find myself getting lost in the `bob` section of writing this word, this might be a decision that has already been made by the broader web community

Polaris - Combobox
https://polaris.shopify.com/components/selection-and-input/combobox

MDN aria thing - Comboboxes
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role

Reply:
Fair!